assets/stylesheets/shared/_colors.scss

```diff
@@ -1,6 +1,6 @@
 
 // The Muriel Color Palette
-@import '../../../node_modules/color-studio/dist/colors';
```

Comment:
the ~ only works from webpack. `node-sass` doesn't understand it. try `npm run prebuild-css`

Reply:
Fixed!